docs/filter.rst

```diff
@@ -16,59 +16,72 @@ Selectors
-    | ``type:osm-type``      | | matches all entities of the      | ``type:node``          |
-    |                        | | given osm type                   |                        |
-    +------------------------+------------------------------------+------------------------+
-    | ``id:osm-id``          | | matches all entities with the    | ``id:1234``            |
-    |                        | | given osm id [1]_                |                        |
-    +------------------------+------------------------------------+------------------------+
-    | ``id:osm-type/osm-id`` | | matches the entity with the given| ``id:node/1234``       |
-    |                        | | osm type and id                  |                        |
-    +------------------------+------------------------------------+------------------------+
-    | ``id:(id list)``       | | matches all entities with the    | ``id:(1, 42, 1234)``   |
-    |                        | | given osm ids [1]_               |                        |
-    +------------------------+------------------------------------+------------------------+
-    | ``id:(id list)``       | | matches all entities with the    | ``id:(node/1, way/3)`` |
-    |                        | | given osm types and ids          |                        |
-    +------------------------+------------------------------------+------------------------+
-    | ``id:(id range)``      | | matches all entities with an id  | ``id:(1 .. 9999)``     |
-    |                        | | matching the given id range [2]_ |                        |
-    +------------------------+------------------------------------+------------------------+
-    | ``geometry:geom-type`` | | matches anything which has a     | ``geometry:polygon``   |
-    |                        | | geometry of the given type       |                        |
-    |                        | | (point, line, polygon, or other) |                        |
-    +------------------------+------------------------------------+------------------------+
-    | ``area:(from..to)``    | | matches features with a geometry | ``area:(1.0 .. 1E6)``  |
-    |                        | | having an area (measured in m²)  |                        |
-    |                        | | in the given range [2]_          |                        |
-    +------------------------+------------------------------------+------------------------+
-    | ``length:(from..to)``  | | matches features with a geometry | ``length:( .. 100)``   |
-    |                        | | having a length (measured in m)  |                        |
-    |                        | | in the given range [2]_          |                        |
-    +------------------------+------------------------------------+------------------------+
+    +-------------------------+------------------------------------+------------------------+
```

Comment:
"unfortunately", I had to expand the whole table because `changeset:(from..to)` didn't fit in the first column. :see_no_evil: